src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -170,10 +158,16 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
     return {
       scheduleDate: day,
       daysEvents: showPersonalisedSchedule
-        ? eventsFilledWithPriority.filter((event) => event.isSaved)
+        ? filterRelatedEvents
+          ? eventsFilledWithPriority.filter(
+              (event) =>
+                event.isSaved &&
+                event.venueId === currentVenue?.id?.toLowerCase()
+            )
+          : eventsFilledWithPriority.filter((event) => event.isSaved)
         : filterRelatedEvents
         ? eventsFilledWithPriority.filter(
-            (event) => event.room === venueRoomTitle
+            (event) => event.venueId === currentVenue?.id?.toLowerCase()
```

Comment:
Hard to follow which expressions are which operands of the multiple `?:` operators. Recommend extracting the values into manageable variables before `return`

Reply:
fix in fc6549ba7fe1cff05c89fdbbd261def4decd3b54